### PR TITLE
[Image_Picker] Fix example

### PR DESCRIPTION
**Pull request by BugsBunnyBR**
## Description
This PR updates the image picker plugin dependency for video_player and updates the video player logic to make it work (the current implementation fails to show the video).

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [X] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [X] My PR includes unit or integration tests for *all* changed/updated/fixed behaviors (See [Contributor Guide]).
- [X] All existing and new tests are passing.
- [X] I updated/added relevant documentation (doc comments with `///`).
- [X] The analyzer (`flutter analyze`) does not report any problems on my PR.
- [X] I read and followed the [Flutter Style Guide].
- [X] The title of the PR starts with the name of the plugin surrounded by square brackets, e.g. [shared_preferences]
- [X] I updated pubspec.yaml with an appropriate new version according to the [pub versioning philosophy].
- [X] I updated CHANGELOG.md to add a description of the change.
- [X] I signed the [CLA].
- [X] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Does your PR require plugin users to manually update their apps to accommodate your change?

- [ ] Yes, this is a breaking change (please indicate a breaking change in CHANGELOG.md and increment major revision).
- [X] No, this is *not* a breaking change.

### CODE REVIEW

File: packages/image_picker/example/lib/main.dart [hunk 1]
```diff
@@ -68,13 +70,14 @@ class _MyHomePageState extends State<MyHomePage> {
   }
 
   void _onVideoControllerUpdate() {
-    setState(() {});
+    // setState(() {});
```
[the remark this answers is not in the document]
Comment (BugsBunnyBR) [reply packages/image_picker/example/lib/main.dart]: done.

**Comment by BugsBunnyBR** [comment]
> Playing video seems failing on iOS with this patch. Could be a bug in the plugin itself too.

I don't have access to an iPhone, but I did test in the simulator. I was not able to see the video, but I could listen to it.
I ended doing some more cleaning in my last commit, if you could take a look.

**Comment by cyanglaz** [comment]
It seems like the video is still not playing on iOS with this patch. I am adding this PR to backlog. Maybe you can create an issue and refer this PR to it. And we will prioritize the PR based on the thumb-ups of the issue?

**Comment by cyanglaz** [comment]
Actually I have found the issue. The video player dependency wasn't updated in the example app. I updated it in the same PR and going to merge it.
Thanks again for the contribution!

**Comment by BugsBunnyBR** [comment]
@cyanglaz It looks like there is some issue with cirrus. Could you help me with that?

**Comment by BugsBunnyBR** [comment]
@cyanglaz it is green.

**Pull request merged**